server/test/unit/common/config/test_external_config.py

```diff
@@ -0,0 +1,107 @@
+import os
+import tempfile
+import unittest
+from unittest.mock import patch
+
+import requests
+
+from server.common.config.app_config import ExternalConfig
+from server.common.config.app_config import AppConfig
+from server.test import test_server, FIXTURES_ROOT
+
+
+class TestExternalConfig(unittest.TestCase):
+    def test_type_convert(self):
```

Comment:
Add a negative test case? What circumstances under which will this function fail?

Reply:
The function doesn't fail.  It will return the input value if it doesn't convert to anything else.